docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+services:
+  redis:
+    container_name: redis
```

Comment:
Let's clean this up a little, we don't need a lot of things like, `container_name`, `volumes`, `entrypoint`.